experiments/mock_service.rb

```diff
@@ -182,31 +211,8 @@ def base_latency(endpoint_index)
         @endpoint_latencies[endpoint_index]
       end
 
-      # Check if a specific endpoint would timeout with current settings
```

Comment:
these three function calls were not called anywhere. Just AI fluff. Removed them